nixos/modules/services/logging/packetbeat.nix

```diff
@@ -0,0 +1,185 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.packetbeat;
+
+  packetbeatYml = pkgs.writeText "packetbeat.yml" ''
```

Comment:
See comment below about a `settings` option which could replace this.

Reply:
I will look into that RFC, I've added the configFile option, that short-circuits the config options in the module, and lets the user handle the configuration file as they please.